Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:current-slim
+
+WORKDIR /usr/src/index
+COPY package.json .
+RUN npm install
+EXPOSE 3000
+CMD [ "npm", "start" ]
+COPY . .
```

Comment:
ENTRYPOINT [ "npm", "start" ]
there is a small difference between `ENTRYPOINT` and `CMD` see here: https://docs.docker.com/engine/reference/builder/#entrypoint

another thing, you don't need to `COPY` twice, can be just:
```
COPY . .
RUN npm install
EXPOSE 3000
ENTRYPOINT [ "npm", "start" ]
```